### ISSUE TITLE
Add Gradient Descent algorithm

### ISSUE THREAD

**Issue by TarunTomar122** [OptimaNiko/ML-algorithms#5]
A simple but very effective machine learning algorithm - Gradient Descent.

**Comment by TarunTomar122** [comment]
I would like to work on this if ok?

**Comment by OptimaNiko** [comment]
@TarunTomar122 Yes, you are more than welcome to work on Gradient Descent. Make sure you submit before October 31 if you want to take part in Hacktoberfest

**Comment by TarunTomar122** [comment]
Sure i will do it today only!

**Comment by OptimaNiko** [comment]
@TarunTomar122 Please update contributors.md file
